docker-compose.yml

```diff
@@ -11,6 +11,7 @@ services:
             - container_only_node_modules:/studio-frontend/node_modules
         ports:
             - "18011:18011"
+            - "9229:9229"
```

Comment:
Chatted with @fysheets to confirm: this should be fine to add, given that we don't use this docker setup outside of development contexts.